client/app/scripts/liveblog-settings/styles/general-settings.less

```diff
@@ -0,0 +1,20 @@
+@PADDING: 20px;
+.general-settings {
```

Comment:
It seems similar to `client/app/scripts/liveblog-edit/styles/settings.less`, we should factorize them.